shared/src/main/scala/mlscript/MLParser.scala

```diff
@@ -204,7 +204,7 @@ class MLParser(origin: Origin, indent: Int = 0, recordLocations: Bool = true) {
           ms.collect { case R(md) => md }, ms.collect{ case L(md) => md }, Nil)
       }
       case (k @ Als, id, ts) => "=" ~ ty map (bod => TypeDef(k, id, ts, bod, Nil, Nil, Nil))
-      case (k @ Nms, id, _) => throw new Exception("Namespaces are not supported yet.")
+      case (k @ Nms, _, _) => assert(false, "Namespaces are not supported yet."); null
```

Comment:
WHAT! Uuuuugh, no. `null` is an absolute anti-pattern. At this point just use `throw new AssertionError(...)`.